src/operators.jl

```diff
@@ -27,6 +27,12 @@ for (op, fun) in [(:+, :plusconstant), (:-, :minusconstant), (:*, :multconstant)
     @eval Base.$op(p::APL, α) = $fun(p, α)
     @eval Base.$op(α, p::APL) = $fun(α, p)
 end
+# Special case AbstractArrays of APLs
+for op in [:+, :-, :*]
+    @eval Base.$op(p::APL, A::AbstractArray{<:APL}) = map(f -> $op(p, f), A)
```

Comment:
I know, but on the other hand we cannot constrain the methods other to avoid the case in #104. StaticArrays also still defines the scalar * array ops.

Reply:
great idea. I added it.